civis/tests/test_cli.py

```diff
@@ -81,9 +81,39 @@ def test_blank_output(mock_session):
     # is blank.
     session_context = mock_session.return_value.__enter__.return_value
     session_context.request.return_value.json.side_effect = ValueError()
+    session_context.request.return_value.status_code = 200
 
     op = {"parameters": []}
-    invoke("WIBBLE", "/wobble/wubble", op)
+
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        invoke("WIBBLE", "/wobble/wubble", op)
```

Comment:
For my own understanding, did the `invoke` command raise a `SystemExit` previously that pytest didn't count as an error such that the `pytest.raises` wasn't necessary?  That is, the `pytest.raises` is just needed so that the test can check `.value.code`, right?

Reply:
I don't think `invoke` was raising `SystemExit` previously. See for example the [failing tests](https://travis-ci.com/github/civisanalytics/civis-python/jobs/328391483) from the first commit in this PR. Now I'm kind of curious what happens under the hood when a python program ends without a `sys.exit()` but a quick bit of googling didn't turn up anything ...